example_scenes.py

```diff
@@ -1,6 +1,8 @@
 from manimlib import *
```

Comment:
Is this an opensource library?

Reply:
smoke 1/31/2022

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

Reply:
test

Reply:
test

Reply:
test

Reply:
test

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](http://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

Reply:
test 3/31/2022 exp 4

Reply:
test 4/18/2022

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

Reply:
test

Reply:
automation test data 
A best practice is to always define default timezone of your application in config/application.rb.

Date.today and Time.now always gives the local date and time of the machine, in the machine’s timezone. It makes sense to use Time.zone.now and Time.zone.today to get around the conflicts between timezones on development machines and production servers.

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Readable, Secure, Brittle, Maintainable

Reply:
test

Reply:
test

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Efficient

Reply:
test1